about.html

```diff
@@ -0,0 +1,144 @@
+                    <a href="#" class="nav-list">Sponsor</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-list">News</a>
+                </li>
+                <button type="button" id="nav-btn">Join Now</button>
+            </ul>
+        </nav>
+    </header>
+    <div id="hamburger">
+        <a href="#" id="close">
+            <i id="close-icon" class="material-icons">close</i>
+        </a>
+        <ul id="hamburger-ul">
+            <li>
+                <a class="link" href="./index.html"> Home</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Program</a>
+            </li>
+            <li>
+                <a class="link" href="./about.html"> About</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Sponsor</a>
+            </li>
+            <li>
+                <a class="link" href="#"> News</a>
+            </li>
+        </ul>
+    </div>
+    <main>
+        <section id="about-intro">
+            <h3>"Hello! Data Scientists"</h3>
+            <h1>AI & BIG DATA FOR A SMARTER FUTURE EXHIBITION 2023</h1>
+            <P>
+                AI & Big Data Exhibition brings together the community of experts,
+                academics, and activists who showcase the next generation enterprise 
+                technologies and strategies from the world of Data Science and AI for 
+                a smarter future.
+            </P>
+            <div id="contact">
+                <p>Please contact us per Email for any future questions about AI & Big Data Exhibition 2023!</p>
+                <a href="#">ai-bigdata2023@rdnairobi.org</a>
+            </div>
+        </section>
+        <section id="about-logo-section">
+            <div class="logo-exhibition">
+                <h3>AI & Big Data for a Smarter Future Exhibition 2023 Logo</h3>
+                <hr>
+                <p>The logo for AI & Big Data Exhibition was decided through the logo competition from 8. June 2022 to 7. July 2022.</p>
+            </div>
+            <div class="logo">
+                <img src="images/iCON.png" alt="Logo Icon" />
+            </div>
+        </section>
+        <section id="past-events">
+            <div class="past-events-top">
+                <h3>See the past AI & Big Data Exhibitions</h3>
+                <hr>
+                <p>Take a look at AI & Big Data Exhibitions that took place in Buenos Aires and in Warsaw.</p>
+            </div>
+            <div class="past-events-details">
+                <section id="past-event-buenos" class="single-event">
+                    <h3>2017</h3>
+                    <p>AI & Big Data Exhibition in Buenos Aires</p>
+                </section>
+                <section id="past-event-warsaw" class="single-event">
+                    <h3>2020</h3>
+                    <p>AI & Big Data Exhibition in Warsaw</p>
+                </section>
+            </div>
+        </section>
```

Comment:
- [ ] Please make sure that your image background color and opacity/filter color (Orange) and the exact height and width as the same as defined in the original design. As a developer is important to pay attention to details :smile: 

#### Original design
![Screenshot from 2022-12-02 18-18-19](https://user-images.githubusercontent.com/6615394/205359942-3d01c132-409e-4554-9c9d-48645f9b15a0.png)

#### Your design
![Screenshot from 2022-12-02 18-22-38](https://user-images.githubusercontent.com/6615394/205360273-71bad182-0eaa-4580-8213-f6058cc48a7c.png)